src/collective/upload/upgrades/v3/__init__.py

```diff
@@ -1,6 +1,36 @@
 # -*- coding: utf-8 -*-
 from collective.upload.config import PROJECTNAME
 from collective.upload.logger import logger
+from plone import api
+
+
+FIX_CSS = {
+    'old': '++resource++collective.upload/css/bootstrap.min.css',
+    'new': '++resource++collective.js.bootstrap/css/bootstrap.min.css'
+}
+FIX_JS = {
+    'old': '++resource++collective.upload/vendor/bootstrap.min.js',
+    'new': '++resource++collective.js.bootstrap/js/bootstrap.min.js'
+}
+
+
+def fix_resources_references(setup_tool):
```

Comment:
@rodfersou you should instead remove bootstrap resources and guarantee collective.js.bootstrap is installed, if needed.